Bottom: 42e05e1f7f0c46ae7d58d00aac9e52697516f4b6
Top:    49fb0d8121f169fb2458ae7eb077a72f3aebd5f7
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-28 23:59:58 +0100

Refresh of hid-gerber-don-t-use-the-core-

---

diff --git a/src/hid/gerber/gerber.c b/src/hid/gerber/gerber.c
index 7573d24..ec32be5 100644
--- a/src/hid/gerber/gerber.c
+++ b/src/hid/gerber/gerber.c
@@ -529,7 +529,7 @@ gerber_expose (HID * hid, BoxType *drawn_area, void *item)
         DrawLayerGroup (group, drawn_area);
     }
 
-  count_holes (drawn_area, &nplated, &nunplated);
+  CountHoles (drawn_area, &nplated, &nunplated);
 
   if (nplated && set_layer ("plated-drill", SL (PDRILL, 0)))
     DrawHoles (true, drawn_area);
